src/PostgREST/Config.hs

```diff
@@ -248,6 +266,9 @@ readOptions = do
           |## unix socket location
           |## if specified it takes precedence over server-port
           |# server-unix-socket = "/tmp/pgrst.sock"
+          |## unix socket file mode
+          |## if server-unix-socket is used, but an invalid mode or none is provided, 755 is applied
+          |# server-unix-socket-mode = "777"
```

Comment:
Example value could be `server-unix-socket-mode = "755"` to better signal the default.

Also, the comment about `invalid mode` should be removed since now we crash when the value is invalid.